pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -599,7 +603,7 @@
     "otp" = ps: with ps; [ pyotp ];
     "ovo_energy" = ps: with ps; [ ]; # missing inputs: ovoenergy
     "owntracks" = ps: with ps; [ pynacl aiohttp-cors hass-nabucasa paho-mqtt ];
-    "ozw" = ps: with ps; [ aiohttp-cors paho-mqtt ]; # missing inputs: python-openzwave-mqtt[mqtt-client]
+    "ozw" = ps: with ps; [ aiohttp-cors homeassistant-pyozw paho-mqtt pydispatcher python-openzwave-mqtt ];
```

Comment:
Wishlist: Make `parse-requirements.py` discover that `python-openzwave-mqtt[mqtt-client]` depends on `asyncio-mqtt`.
Maybe we can add a few of these by hand for now (in `parse-requirements.py`, not `component-packages.nix`).

Reply:
So you want it to also show missing transitive dependencies? How would we know this?

Reply:
We would also have to look at their `setup.py`s. How does `pip` check it when you install a package requiring `python-openzwave-mqtt[mqtt-client]`?

Reply:
Pretty sure that gets complicated fast, unless we can reuse the logic pip uses. Because `setup.py` might contain code that evaluates the list of requirements during runtime. Or they might be given in `requirements.txt` format, or `pyproject.toml`.

Reply:
Yes definitely. And since I have no idea of whether you can hook into pip's mechanism, I suggested we do it by hand for a few exceptions.
We might want to have a dict mapping e.g. `'python-openzwave-mqtt[mqtt-client]'` to `['asyncio-mqtt']` so that whenever a key doesn't exist in the dict, we can print a warning that some dependencies should be added manually.

Reply:
> Anyway I don't really understand why we need this, besides estimating how much effort it is to get a component running.

Just so that people using e.g. zwave are not bothered by missing dependencies that they have to add manually.

Reply:
https://github.com/wimglenn/johnnydep looks small enough to either use or learn that pip api from.